[2024-11-20T04:17:36.666+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_metrics.publish.customer_transactions_run manual__2024-11-20T04:15:26.082029+00:00 [queued]>
[2024-11-20T04:17:36.678+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_metrics.publish.customer_transactions_run manual__2024-11-20T04:15:26.082029+00:00 [queued]>
[2024-11-20T04:17:36.679+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-11-20T04:17:36.695+0000] {taskinstance.py:2214} INFO - Executing <Task(DbtRunLocalOperator): publish.customer_transactions_run> on 2024-11-20 04:15:26.082029+00:00
[2024-11-20T04:17:36.700+0000] {standard_task_runner.py:60} INFO - Started process 15912 to run task
[2024-11-20T04:17:36.703+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'customer_metrics', 'publish.customer_transactions_run', 'manual__2024-11-20T04:15:26.082029+00:00', '--job-id', '147', '--raw', '--subdir', 'DAGS_FOLDER/customer_metrics.py', '--cfg-path', '/tmp/tmp2909ky_p']
[2024-11-20T04:17:36.705+0000] {standard_task_runner.py:88} INFO - Job 147: Subtask publish.customer_transactions_run
[2024-11-20T04:17:36.759+0000] {task_command.py:423} INFO - Running <TaskInstance: customer_metrics.publish.customer_transactions_run manual__2024-11-20T04:15:26.082029+00:00 [running]> on host 4cf1ca1acf00
[2024-11-20T04:17:36.857+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='customer_metrics' AIRFLOW_CTX_TASK_ID='publish.customer_transactions_run' AIRFLOW_CTX_EXECUTION_DATE='2024-11-20T04:15:26.082029+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-20T04:15:26.082029+00:00'
[2024-11-20T04:17:36.859+0000] {local.py:202} INFO - Cloning project to writable temp directory /tmp/tmpagzxkpd4 from /opt/airflow/include/dbt/fraud
[2024-11-20T04:17:36.866+0000] {config.py:253} INFO - Using user-supplied profiles.yml at /opt/airflow/include/dbt/fraud/profiles.yml
[2024-11-20T04:17:36.867+0000] {local.py:235} INFO - Trying to run the command:
 ['/home/airflow/.local/bin/dbt', 'run', '--models', 'customer_transactions', '--profiles-dir', '/opt/airflow/include/dbt/fraud', '--profile', 'fraud', '--target', 'dev']
From /tmp/tmpagzxkpd4
[2024-11-20T04:17:36.867+0000] {local.py:236} INFO - Using environment variables keys: dict_keys(['AIRFLOW_CTX_DAG_OWNER', 'AIRFLOW_CTX_DAG_ID', 'AIRFLOW_CTX_TASK_ID', 'AIRFLOW_CTX_EXECUTION_DATE', 'AIRFLOW_CTX_TRY_NUMBER', 'AIRFLOW_CTX_DAG_RUN_ID'])
[2024-11-20T04:17:36.868+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-11-20T04:17:36.869+0000] {subprocess.py:73} INFO - Running command: ['/home/airflow/.local/bin/dbt', 'run', '--models', 'customer_transactions', '--profiles-dir', '/opt/airflow/include/dbt/fraud', '--profile', 'fraud', '--target', 'dev']
[2024-11-20T04:17:36.879+0000] {subprocess.py:84} INFO - Command output:
[2024-11-20T04:17:38.980+0000] {subprocess.py:94} INFO - [0m04:17:38  Running with dbt=1.8.7
[2024-11-20T04:17:40.308+0000] {subprocess.py:94} INFO - [0m04:17:40  Registered adapter: snowflake=1.8.4
[2024-11-20T04:17:40.340+0000] {subprocess.py:94} INFO - [0m04:17:40  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2024-11-20T04:17:42.162+0000] {subprocess.py:94} INFO - [0m04:17:42  Found 3 models, 454 macros
[2024-11-20T04:17:42.163+0000] {subprocess.py:94} INFO - [0m04:17:42
[2024-11-20T04:17:45.747+0000] {subprocess.py:94} INFO - [0m04:17:45  Concurrency: 1 threads (target='dev')
[2024-11-20T04:17:45.747+0000] {subprocess.py:94} INFO - [0m04:17:45
[2024-11-20T04:17:45.754+0000] {subprocess.py:94} INFO - [0m04:17:45  1 of 1 START sql table model AIRBYTE_SCHEMA.customer_transactions .............. [RUN]
[2024-11-20T04:17:48.234+0000] {subprocess.py:94} INFO - [0m04:17:48  1 of 1 OK created sql table model AIRBYTE_SCHEMA.customer_transactions ......... [[32mSUCCESS 1[0m in 2.48s]
[2024-11-20T04:17:48.237+0000] {subprocess.py:94} INFO - [0m04:17:48
[2024-11-20T04:17:48.238+0000] {subprocess.py:94} INFO - [0m04:17:48  Finished running 1 table model in 0 hours 0 minutes and 6.07 seconds (6.07s).
[2024-11-20T04:17:48.282+0000] {subprocess.py:94} INFO - [0m04:17:48
[2024-11-20T04:17:48.282+0000] {subprocess.py:94} INFO - [0m04:17:48  [32mCompleted successfully[0m
[2024-11-20T04:17:48.283+0000] {subprocess.py:94} INFO - [0m04:17:48
[2024-11-20T04:17:48.283+0000] {subprocess.py:94} INFO - [0m04:17:48  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-11-20T04:17:49.547+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-11-20T04:17:49.562+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-11-20T04:17:49.562+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-11-20T04:17:49.569+0000] {local.py:252} INFO - Inlets: []
[2024-11-20T04:17:49.570+0000] {local.py:253} INFO - Outlets: [Dataset(uri='snowflake://qab09154.us-east-1.aws/AIRBYTE_DATABASE.AIRBYTE_SCHEMA.customer_transactions', extra=None)]
[2024-11-20T04:17:49.570+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-11-20T04:17:49.596+0000] {dag.py:3834} INFO - Setting next_dagrun for customer_metrics to 2024-11-20 00:00:00+00:00, run_after=2024-11-21 00:00:00+00:00
[2024-11-20T04:17:49.623+0000] {local.py:371} INFO - [0m04:17:48  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-11-20T04:17:49.629+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py:1469 RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
[2024-11-20T04:17:49.650+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=customer_metrics, task_id=publish.customer_transactions_run, execution_date=20241120T041526, start_date=20241120T041736, end_date=20241120T041749
[2024-11-20T04:17:49.708+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-20T04:17:49.723+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
